nextflow.config

```
@@ -9,10 +9,9 @@
 params {
 
   // Workflow flags
-  // TODO nf-core: Specify your pipeline's command line flags
-  genome = false
-  reads = "data/*{1,2}.fastq.gz"
-  singleEnd = false
+  input = null
+  params.chr = null
```

```suggestion
  input = false
  chr = false
```
we generally set it to `false`, I don't recall now if `null` could generate some problems, but unless you have a reason for it I would say you could set it to `false` instead.